### PR TITLE
Fixed link to auth_test.go in coding-conventions.md

### PR DESCRIPTION
**Pull request by ianchakeres**

### CODE REVIEW

File: contributors/devel/coding-conventions.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ following Go conventions - `stateLock`, `mapLock` etc.
 tests
 
   - Table-driven tests are preferred for testing multiple scenarios/inputs; for
-example, see [TestNamespaceAuthorization](../../test/integration/auth/auth_test.go)
+example, see [TestNamespaceAuthorization](https://github.com/kubernetes/kubernetes/blob/master/test/integration/auth/auth_test.go)
```
Comment (cblecker) [inline contributors/devel/coding-conventions.md]: Can we use "https://git.k8s.io/kubernetes/test/integration/auth/auth_test.go" for this?

Comment (ianchakeres) [inline contributors/devel/coding-conventions.md]: @cblecker - I'm fine with using the path that you mention, it resolves to the URL I specified. Why do you prefer this path?

Note: the path I've used is referenced in a few other spots in the community repo.

Comment (cblecker) [inline contributors/devel/coding-conventions.md]: In the (unlikely) event that the target for that document changes, git.k8s.io is something we control [here](https://github.com/kubernetes/k8s.io/blob/fb20b6aa18d8d6fab882df36ee534bd8208d9900/k8s.io/configmap-nginx.yaml#L205-L215). Not critical to go at the moment and update everything, but if we're fixing broken links, we might as well use it :)

Comment (ianchakeres) [inline contributors/devel/coding-conventions.md]: @cblecker - ok - I updated the three references to auth_test.go.

**Pull request merged**